### PR TITLE
Iframe Storage Selector

### PR DESCRIPTION
**Pull request by ezequielc**
- Iframe Storage Selector
- gulp updates
- Single file selector
- Single folder selector
- notify parent page on file/folder selection
- close modal
- test coverage/refactoring 

Missing authentication functionality before releasing.
@alex-deaconu fyi

### CODE REVIEW

File: web/partials/storage/storage-modal.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div id="storageSelectorModal" ng-controller="FilesListController">
 <div class="modal-header">
-  <button type="button" id="closeButton" class="close" ng-click="dismiss()" data-dismiss="modal" aria-hidden="true">
+  <button type="button" id="closeButton" class="close" ng-click="fileSelectorFactory.cancel()" data-dismiss="modal" aria-hidden="true">
```
Comment (alex-deaconu) [inline web/partials/storage/storage-modal.html]: @ezequielc should also update the `cancel()` function for the `Cancel` button below: https://github.com/Rise-Vision/rise-vision-apps/pull/134/files#diff-2058d9bd0d60eb4f6c9e031026690a41L39. Thanks.

Comment (ezequielc) [inline web/partials/storage/storage-modal.html]: Thanks @alex-deaconu. I will update. Cheers

Comment (ezequielc) [inline web/partials/storage/storage-modal.html]: @alex-deaconu Are we using multiple file selector somewhere? I didn't test it.
And if we are, we should add it here too: https://github.com/Rise-Vision/rise-vision-apps/pull/134/files/b5bdfd7b3ce3969d391af2de2fbb09a04790150b#diff-41bf726360f1477df0b53cee3a15fad3R101
Thanks

Comment (alex-deaconu) [inline web/partials/storage/storage-modal.html]: @ezequielc I'm not sure if we are, but I think we should add it for good measure. Thanks.

Comment (ezequielc) [inline web/partials/storage/storage-modal.html]: Thanks @alex-deaconu I updated it. Please review. Cheers

**Comment by alex-deaconu** [comment]
@ezequielc looks good 👍 . Thanks for fixing. Just staged and tested this branch in RVA. Something's not right and the message is not relayed back. Not sure if this is RVA or Apps related, still investigating. 

We can merge this (and update Widgets in the meantime) while I fix the RVA implementation, or we can hold off. Up to you.

Thanks.

**Comment by ezequielc** [comment]
Thanks @alex-deaconu. I am updating the widgets but staged widgets are using production server, so it is tricky to test without deploying this. 
As deploying won't affect anything, I would suggest to merge this PR. Then I can validate if widgets are receiving the message or not. Do you agree? 
Thanks

**Comment by alex-deaconu** [comment]
@ezequielc makes sense, I'm good with merging. Thanks!

**Comment by ezequielc** [comment]
Thanks @alex-deaconu I will squash and merge. Cheers

**Comment by alex-deaconu** [comment]
👍

**Pull request merged**